// @vivaxy/git/lib/status/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.getRootPath = exports.isInsideWorkTree = exports.isRepositoryRoot = exports.isClean = exports.getHeadCommit = exports.getStagedFiles = void 0;
/**
 * @since 2019-07-31 15:01
 * @author vivaxy
 */
var get_staged_files_1 = require("./get-staged-files");
Object.defineProperty(exports, "getStagedFiles", { enumerable: true, get: function () { return get_staged_files_1.default; } });
var get_head_commit_1 = require("./get-head-commit");
Object.defineProperty(exports, "getHeadCommit", { enumerable: true, get: function () { return get_head_commit_1.default; } });
var is_clean_1 = require("./is-clean");
Object.defineProperty(exports, "isClean", { enumerable: true, get: function () { return is_clean_1.default; } });
var is_repository_root_1 = require("./is-repository-root");
Object.defineProperty(exports, "isRepositoryRoot", { enumerable: true, get: function () { return is_repository_root_1.default; } });
var is_inside_work_tree_1 = require("./is-inside-work-tree");
Object.defineProperty(exports, "isInsideWorkTree", { enumerable: true, get: function () { return is_inside_work_tree_1.default; } });
var get_root_path_1 = require("./get-root-path");
Object.defineProperty(exports, "getRootPath", { enumerable: true, get: function () { return get_root_path_1.default; } });
